### PR TITLE
[VL] Update velox-backend-limitations.md

### PR DESCRIPTION
**Pull request by FelixYBW**
update velox limitation documents

### CODE REVIEW

**Comment by FelixYBW** [comment]
@JkSelf To the ctas and dynamic partition write, will Gluten fallback to Vanilla spark or raise exceptions?

**Comment by JkSelf** [comment]
> @JkSelf To the ctas and dynamic partition write, will Gluten fallback to Vanilla spark or raise exceptions?

@FelixYBW Yes.  All fallback to vanilla spark not throw exceptions.

File: docs/velox-backend-limitations.md [hunk 1]
```diff
@@ -33,79 +39,76 @@ java.io.IOException: Unexpected Bloom filter version number (512)
...
+  .format("parquet")
+  .partitionBy("p")
+  .save(f.getCanonicalPath)
+```
 
-#### Offload native write to velox
+#### CTAS write
 
-We implemented write support by overriding the following vanilla Spark classes. And you need to ensure preferentially load the Gluten jar to overwrite the jar of vanilla spark. Refer to [How to prioritize loading Gluten jars in Spark](https://github.com/oap-project/gluten/blob/main/docs/velox-backend-troubleshooting.md#incompatible-class-error-when-using-native-writer). It should be noted that if the user also modifies the following overriding classes, the user's changes may be overwritten.
+Velox does not create table as select. It may raise exception. e.g.,
```
Comment (JkSelf) [inline docs/velox-backend-limitations.md:97]: Gluten will fall back to vanilla Spark.

File: docs/velox-backend-support-progress.md [hunk 2]
```diff
@@ -9,9 +9,7 @@ Gluten is still in active development. Here is a list of supported operators and
 
 Since the same function may have different semantics between Presto and Spark, Velox implement the functions in Presto category, if we note a different sematics from Spark, then the function is implemented in Spark category. So Gluten firstly will use Velox's spark category, if a function isn't implemented there then refer to Presto category.
 
-The total supported functions' number for [Spark3.3 is 387](https://spark.apache.org/docs/latest/api/sql/) and for [Velox is 204](https://facebookincubator.github.io/velox/functions/coverage.html).
-Gluten supported frequently used 164, shown as below picture.
-![support](./image/support.png)
+The total supported functions' number for [Spark3.3 is 387](https://spark.apache.org/docs/latest/api/sql/), Gluten supported 189 functions now.
```
Comment (JkSelf) [inline docs/velox-backend-support-progress.md:12]: Can we also put the latest function support png here?

**Review by zhouyuan — APPROVED** [review]

**Comment by zhouyuan** [comment]
I'll follow up on the refine.

**Pull request merged**